spec/lucky/params_spec.cr

```diff
@@ -148,4 +169,85 @@ describe Lucky::Params do
       params.nested(:missing).should eq({} of String => String)
     end
   end
+
+  describe "get_file" do
+    it "gets files" do
+      request = build_multipart_request file_parts: {
+        "welcome" => "hello paul"
```

Comment:
Could we make it more clear what each of these represent by doing something like "welcome_file" => "hello-paul.png" or something? What do you think about that?

Reply:
Yeah, that makes a lot of sense. Will rename to be clearer. Could also create `Tempfile` objects to pass in here but thought that might clutter up the specs.